community/modules/compute/gke-topology-scheduler/manifests/schedule-daemon.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: topology-scheduler
+  labels:
+    app: topology-scheduler
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: topology-scheduler
+  template:
+    metadata:
+      labels:
+        app: topology-scheduler
+    spec:
+      tolerations:
+      - key: "node-role.kubernetes.io/control-plane"
```

Comment:
Should we add a toleration similar to kueue to allow it to land on system nodepool ? 

```
tolerations:
      - effect: NoSchedule
        key: components.gke.io/gke-managed-components
        operator: Equal
        value: "true"
```

Reply:
Which nodepool does Topology scheduler gets provisioned on ? By default our blueprints have two nodepools:
* system (Does not allow any thing to get provisioned until we add the toleration i was describing)
* GPU (we only want GPU workload to land here)

So did you add another nodepool to your blueprint for testing ?

Reply:
Followed offline, this testing will be done with full blueprint